daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md

```diff
@@ -6,7 +6,7 @@ weight: 2000
 description: "Use the secret store building block to securely retrieve a secret"
 ---
 
-This guide demonstrates how to use Dapr's secrets API in your code to leverage the [secrets store building block]({{< ref secrets-overview >}}). With the secrets API, you easily retrieve secrets in your application code from a configured secret store.
+Now that you've learned [what the Dapr secrets building block provides]({{< ref secrets-overview >}}), learn how it can work in your service. This guide demonstrates how to call out the secrets API in your code and easily retrieve secrets in your application code from a configured secret store.
 
 <img src="/images/building-block-secrets-management-example.png" width=1000 alt="Diagram showing secrets management of example service">
```

Comment:
This diagram needs updating. Really the example style from the quickstarts should be used, like this https://docs.dapr.io/getting-started/quickstarts/statemanagement-quickstart/ but do one for secrets and add the same diagram back to the secrets quickstart here https://docs.dapr.io/getting-started/quickstarts/secrets-quickstart/ which does not have a diagram.

Also could do with updating this diagram with another PR for state management. https://docs.dapr.io/developing-applications/building-blocks/state-management/howto-get-save-state/